charts/konk/templates/kind-pod.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: {{ include "konk.fullname" . }}-kind-pod
+  labels:
+    {{- include "konk.labels" . | nindent 4 }}
+spec:
+  serviceAccountName: {{ include "konk.serviceAccountName" . }}
+  containers:
+  - name: kind
+    securityContext:
+      {{- toYaml .Values.kind.securityContext | nindent 6 }}
+    image: "{{ .Values.kind.image.repository }}:{{ .Values.kind.image.tag }}"
+    imagePullPolicy: {{ .Values.kind.image.pullPolicy }}
+    command:
+      - bash
+    args:
+      - /kind/start.sh
+    ports:
+      - name: https
+        containerPort: 443
+        protocol: TCP
+    resources:
+      {{- toYaml .Values.kind.resources | nindent 6 }}
+    volumeMounts:
+    - mountPath: /kind/
+      name: certs-configmap
+      readOnly: true
+  volumes:
+  - name: certs-configmap
```

The naming here is a little misleading. I'd probably call it `scripts`
```suggestion
    volumeMounts:
    - mountPath: /scripts/
      name: scripts
      readOnly: true
  volumes:
  - name: scripts
```

---

Done